### PR TITLE
Rename adaptors to ledgers and connections

### PR DESCRIPTION
**Pull request by ryanwjackson**
Note, this is a breaking change, so we should definitely do a minor bump.

### CODE REVIEW

**Comment by ryanwjackson** [comment]
@SeanBolt can you QA against Platform tests once you have made the naming changes?  Just want to test as much as possible.

File: README.md [hunk 1]
```diff
@@ -32,16 +32,16 @@ $ gem install ledger_sync
...
 
 The code may look like the following:
 
 ```ruby
-# First we create an adaptor, which is our connection to a ledger.
+# First we create an connection, which is our connection to a ledger.
 # Each ledger may require different keys, so check the
 # documentation below for specifics.
-adaptor = LedgerSync::Adaptors::QuickBooksOnline::Adaptor.new(
+connection = LedgerSync::Ledgers::QuickBooksOnline::Ledger.new(
```
Comment (jozefvaclavik) [inline README.md:44]: I'm not sure I'm fan of this naming convention. Why not to name it `LedgerSync::Ledgers::QuickBooksOnline::Connection.new(`

File: README.md [hunk 2]
```diff
@@ -145,30 +145,30 @@ customer_class_2.resource_attributes[:bar].type # => #<LedgerSync::Type::Boolean
...
 You can now use these custom resources in operations that require custom attributes.
 
-### Adaptors
+### Ledgers
 
-Adaptors are ledger-specific ruby objects that contain all the logic to authenticate to a ledger, perform ledger-specific operations, and validate resources based on the requirements of the ledger.  Adaptors contain a number of useful objects:
+Ledgers are ledger-specific ruby objects that contain all the logic to authenticate to a ledger, perform ledger-specific operations, and validate resources based on the requirements of the ledger.  Ledgers contain a number of useful objects:
 
-- adaptor
+- connectn
```
Comment (SeanBolt) [inline README.md:152]: This should be `client`

File: lib/ledger_sync/ledgers/quickbooks_online/client.rb [hunk 3]
```diff
@@ -4,9 +4,9 @@
 require_relative 'dashboard_url_helper'
 
 module LedgerSync
-  module Adaptors
+  module Ledgers
     module QuickBooksOnline
-      class Adaptor < LedgerSync::Adaptors::Adaptor
+      class Client < Ledgers::Client
```
Comment (SeanBolt) [inline lib/ledger_sync/ledgers/quickbooks_online/client.rb:9]: Does this still work without the LedgerSync module reference?

File: spec/ledgers/netsuite/account/searcher_spec.rb [hunk 4]
```diff
@@ -4,7 +4,7 @@
 
 support 'netsuite/shared_examples'
 
-RSpec.describe LedgerSync::Adaptors::NetSuite::Vendor::Searcher do
+RSpec.describe LedgerSync::Ledgers::NetSuite::Account::Searcher do
```
Comment (SeanBolt) [inline spec/ledgers/netsuite/account/searcher_spec.rb:7]: Why do we move some of these specs from `Vendor`/`Currency` to `Account`?

File: lib/ledger_sync/ledgers/quickbooks_online/client.rb (hunk 3 above)
Comment (ryanwjackson) [reply lib/ledger_sync/ledgers/quickbooks_online/client.rb:9]: Up!

**Pull request merged**